Cutelyst/Plugins/Authentication/authentication.cpp

```diff
@@ -57,13 +57,16 @@ void Authentication::addRealm(Cutelyst::AuthenticationRealm *realm, const QStrin
 {
     Q_D(Authentication);
     realm->setObjectName(name);
+    realm->setName(name);
     d->realms.insert(name, realm);
     d->realmsOrder.append(name);
 }
 
 void Cutelyst::Authentication::addRealm(Cutelyst::AuthenticationStore *store, Cutelyst::AuthenticationCredential *credential, const QString &name)
 {
-    addRealm(new AuthenticationRealm(store, credential, this), name);
+    AuthenticationRealm *realm = new AuthenticationRealm(store, credential, this);
```

Comment:
Please just change this to auto realme = new Auth...
The rule I use is use auto for complex types like iterators, and if the type is explicit at right like in this case, thanks!